_rules/SC1-1-1-accessible-name-is-image-file-name.md

```diff
@@ -0,0 +1,81 @@
+---
+name: image accessible name is image file name
+description: |
+  This rule checks that `img` elements that use their source file name as their accessible name do so without loss of infomation to the user.
+
+success_criterion:
+- 1.1.1 # Non-Text Content
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Bryn Anderson
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to any HTML `img` element that is [exposed to assistive technologies](#exposed-to-assistive-technologies), where the [accessible name](#accessible-name) value exactly matches the file name specified in the `src` attribute.
+
+**Note**: URL paramters, case sensitivey, forward and trailing whitespace should be ignored. For example `src="/foo/bar.jpg?baz "` has a file name "bar.jpg".
```

Comment:
"URL paramters" -> "query string".

Reply:
Should this be a definition for a term like "matching filename"?